content/docs/ref/pr.md

```diff
@@ -32,27 +34,7 @@ Any [generic option](/doc/ref) in addition to:
 cml pr "."
 ```
 
-### Automatically merge GitHub pull requests
-
-```yaml
-on: pull_request
-jobs:
-  cml:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: iterative/setup-cml@v1
-      - name: Generate data
-        run: echo "Hello World" > output.txt
```

Comment:
static output doesn't make sense to PR. Maybe something like

```suggestion
        run: date > output.txt
```

Reply:
I use `date > file` in all my internal tests. 😄 Indeed, it makes more sense.